tools/tests/stepper/test.js

```diff
@@ -94,9 +94,126 @@ describe('Stepper Component', () => {
     });
 
     it('Should select previous option on click', async () => {
-        await loadStepper(STEPPER_OPTIONS_SELECTED);
+        await loadStepper(STEPPER_OPTIONS_SELECTED, STEPPER_OPTIONS_SELECTED[1].name);
         const stepper = document.querySelector('gameface-stepper');
         stepper.leftButton.dispatchEvent(new Event('click'));
+        assert.equal(stepper.value, STEPPER_OPTIONS_SELECTED[0].name);
+    });
+
+    it('Should not change items array with when its value is invalid', async () => {
+        await loadStepper(STEPPER_OPTIONS, STEPPER_OPTIONS[1].name);
+        const stepper = document.querySelector('gameface-stepper');
+        stepper.items = undefined;
+        assert.equal(stepper.value, STEPPER_OPTIONS[1].name);
+        assert.equal(stepper.selectedIndex, 1);
+    });
+
+    it('Should change items array where current value is invalid', async () => {
+        await loadStepper(STEPPER_OPTIONS);
+        const stepper = document.querySelector('gameface-stepper');
+        stepper.items = ['Option 0', 'Option 2', 'Option 3'];
+        assert.equal(stepper.value, 'Option 0');
+        assert.equal(stepper.selectedIndex, 0);
+    });
+
+    it('Should change items array where current value is valid', async () => {
+        await loadStepper(STEPPER_OPTIONS);
+        const stepper = document.querySelector('gameface-stepper');
+        stepper.items = ['Option 0', ...stepper.items];
         assert.equal(stepper.value, STEPPER_OPTIONS[0].name);
+        assert.equal(stepper.selectedIndex, 1);
+    });
+
+    it('Should not change value of the stepper dynamically when it is invalid', async () => {
+        await loadStepper(STEPPER_OPTIONS, STEPPER_OPTIONS[1].name);
+        const stepper = document.querySelector('gameface-stepper');
+        stepper.value = 'invalid';
+        assert.equal(stepper.value, STEPPER_OPTIONS[1].name);
+        assert.equal(stepper.selectedIndex, 1);
+    });
+
+    it('Should change value of the stepper dynamically', async () => {
+        await loadStepper(STEPPER_OPTIONS, STEPPER_OPTIONS[1].name);
+        const stepper = document.querySelector('gameface-stepper');
+        stepper.value = STEPPER_OPTIONS[0].name;
+        assert.equal(stepper.value, STEPPER_OPTIONS[0].name);
+        assert.equal(stepper.selectedIndex, 0);
     });
 });
+
+
+/* global engine */
+/* global setupDataBindingTest */

```

Comment:
I think I have somewhat "incorrectly" or inconveniently used this "global" to ignore the eslint messages and I saw that it can be added in the **package.json** in `eslintConfig` -> "globals" object so this `/* global setupDataBindingTest */` is no longer required everywhere.
Can you add it? Location: GameUIComponents_Public\tools\tests\package.json
Most likely `engine` can be added too.

Reply:
Nice! <3 I fixed it everywhere. Thanks!